runtime/vm/jfr.cpp

```diff
@@ -804,6 +823,90 @@ jfrExecutionSampleCallback(J9VMThread *currentThread, IDATA handlerKey, void *us
 	jfrExecutionSample(currentThread, currentThread);
 }
 
+void
+jfrCPULoad(J9VMThread *currentThread)
+{
+	PORT_ACCESS_FROM_VMC(currentThread);
+	OMRPORT_ACCESS_FROM_J9PORT(PORTLIB);
+
+	omrthread_process_time_t currentProcCPUTimes = {0};
+	intptr_t processTimeRC = omrthread_get_process_times(&currentProcCPUTimes);
+
+	J9SysinfoCPUTime currentSysCPUTime = {0};
+	intptr_t sysTimeRC = omrsysinfo_get_CPU_utilization(&currentSysCPUTime);
+
+	if ((0 == processTimeRC) && (0 == sysTimeRC)) {
+		J9JFRCPULoad *jfrEvent = (J9JFRCPULoad *)reserveBuffer(currentThread, sizeof(J9JFRCPULoad));
+		if (NULL != jfrEvent) {
+			initializeEventFields(currentThread, (J9JFREvent *)jfrEvent, J9JFR_EVENT_TYPE_CPU_LOAD);
+
+			JFRState *jfrState = &currentThread->javaVM->jfrState;
+			uintptr_t numberOfCpus = j9sysinfo_get_number_CPUs_by_type(J9PORT_CPU_ONLINE);
+			int64_t currentTime = j9time_nano_time();
+			omrthread_process_time_t *prevProcCPUTimes = &jfrState->prevProcCPUTimes;
+			int64_t *prevProcTimestamp = &jfrState->prevProcTimestamp;
```

Comment:
Suggest these locals only make the code harder to follow; please remove them (i.e. just use `jfrState->prevProcTimestamp` instead of `*prevProcTimestamp`).